panel/lxqtpanelapplication.cpp

```diff
@@ -44,6 +44,28 @@ LXQtPanelApplicationPrivate::LXQtPanelApplicationPrivate(LXQtPanelApplication *q
 }
 
 
+ILXQtPanel::Position LXQtPanelApplicationPrivate::computeNewPanelPosition()
```

Comment:
@palinek Thanks for the heads up. Only having one screen, completely forgot about it.